tests/test-verify-volume-md.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+  # Only when volume is raid
+- name: check RAID options
+  block:
+
+    - name: get information about RAID
+      command: "mdadm --detail {{ storage_test_volume._device }}"
```

Comment:
This should be `_raw_device` in case it is encrypted.

Reply:
Fixed: Changed to `_raw_device`